src/components/comment-form/comment-form.scss

```diff
@@ -0,0 +1,37 @@
+@use "../../mixins/ms";
+@use "../../compiled/tokens/scss/size";
+
+.c-comment-form {
+  display: grid;
+  grid-gap: ms.step(2, 1rem) ms.step(4, 1rem);
+  grid-template-columns: 1fr 1fr;
+}
+
+.c-comment-form > * {
+  grid-column: span 2;
+}
+
+@media (min-width: size.$breakpoint-s) {
+  .c-comment-form__name,
+  .c-comment-form__email {
+    grid-column: span 1;
+  }
+}
+
+.c-comment-form__subscribe {
+  display: flex;
+
+  & > input {
+    margin-right: ms.step(-2, 1rem);
+  }
+}
+
+.c-comment-form__reply-buttons {
+  display: grid;
+  grid-gap: ms.step(-2, 1rem);
+  grid-template-columns: 1fr 1fr;
+
+  @media (min-width: size.$breakpoint-s) {
+    margin-right: auto;
+  }
+}
```

Comment:
Could this leverage the Button Group stuff somehow? #1240

Reply:
I think it might be easiest to do that in a follow up PR. I made a sticky note so I won't forget 🗒️

Reply:
@tylersticka The button group object doesn't force the buttons to be the same width regardless of text length, but in https://github.com/cloudfour/cloudfour.com-patterns/issues/1216 it shows them being the same width.

I could:

1. Update button group pattern so the buttons don't grow/shrink based on text length
2. Let the buttons in comment form grow/shrink based on text length
3. Not use button group, just use the CSS above ☝️
4. Add a modifier to the button group pattern with the CSS above ☝️ 

What do you think?

Reply:
Also, should the button group pattern have the "full width on small screens" behavior shown here:

![](https://user-images.githubusercontent.com/24928337/118171756-6eecf000-b3e0-11eb-9bd8-263acb88b685.jpg)

Reply:
> 4. Add a modifier to the button group pattern with the CSS above ☝️

This one sounds good to me

> Also, should the button group pattern have the "full width on small screens" behavior shown here:

Probably, potentially as a modifier?